tgui/packages/tgui/components/DepartmentPane.tsx

```diff
@@ -4,15 +4,12 @@ import { Box } from './Box';
 import { Stack } from './Stack';
 
 export interface BaseDepartmentInfo {
-  name: string;
   color: string;
-  jobs: BaseJobInfo[];
+  jobs: { [x: string]: BaseJobInfo };
 }
 
-// Very loose because the job info required can vary, and this component doesn't require any particular data from it.
-export interface BaseJobInfo {
-  name: string;
-}
+// Empty because the job info required can vary, and this component doesn't require any particular data from it.
+export interface BaseJobInfo {}
```

Comment:
1. Use `type` instead.
2. BaseJobInfo is a completely pointless type, if it's supposed to be empty. Type system in TypeScript isn't like in Java. I suggest remove it.
3. Use `Record<string, BaseJobInfo>` instead of `{ [x: string]: BaseJobInfo }`.